### PR TITLE
Add example laplacian matrix 

### PR DESCRIPTION
**Pull request by paulitapb**

### CODE REVIEW

File: networkx/linalg/laplacianmatrix.py [hunk 1]
```diff
@@ -46,6 +46,21 @@ def laplacian_matrix(G, nodelist=None, weight="weight"):
...
     normalized_laplacian_matrix
     laplacian_spectrum
+
+    Examples
+    --------
+    For graphs with multiple connected components, L is permutation-similar
+    to a block diagonal matrix where each block is the respective Laplacian
+    matrix for each component.
+
+    >>> G = nx.graph_atlas(26) #This graph from the Graph Atlas has 2 connected components.
```
Comment (rossbar) [inline networkx/linalg/laplacianmatrix.py:56]: Similar comment here to the other PR - let's use explicit examples for the input Graph

File: networkx/linalg/laplacianmatrix.py [hunk 2]
```diff
@@ -46,6 +46,21 @@ def laplacian_matrix(G, nodelist=None, weight="weight"):
...
+    >>> print(nx.laplacian_matrix(G).todense())
```
Comment (rossbar) [inline networkx/linalg/laplacianmatrix.py:57]: Using `toarray` would be preferable to `todense` here - we can also then get rid of the `print` and just rely on the array repr.

File: networkx/linalg/laplacianmatrix.py (hunk 1 above)
Comment (paulitapb) [reply networkx/linalg/laplacianmatrix.py:56]: I'll do it

File: networkx/linalg/laplacianmatrix.py (hunk 2 above)
Comment (paulitapb) [reply networkx/linalg/laplacianmatrix.py:57]: great!

Comment (paulitapb) [reply networkx/linalg/laplacianmatrix.py]: > Using `toarray` would be preferable to `todense` here - we can also then get rid of the `print` and just rely on the array repr.

Without the `print` there are some docs tests that failed due to differences in some OS. On Windows at the end of the array is added `dtype=int32)`  but not on Ubuntu. Should I add the print back?

Comment (rossbar) [reply networkx/linalg/laplacianmatrix.py]: Interesting - I wasn't expecting that but I suppose makes sense given windows integer types.

I suppose the two options are to add the `print` back or add an explicit cast, e.g. `toarray().astype(np.int64)`. Neither is ideal but both should be fine, so whatever you think is best!

Comment (rossbar) [reply networkx/linalg/laplacianmatrix.py]: The platform-specific printing is likely related to something in numpy itself, see numpy/numpy#22586.

In the meantime, the printing workaround seems fine to me!

**Review by rossbar — APPROVED** [review]
Thanks @paulitapb , just one code style nit that I took the liberty of pushing up (I think it should've been caught by `blacken-docs` in CI?). LGTM!

**Review by MridulS — APPROVED** [review]

**Pull request merged**